### PR TITLE
Modal card

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
## Portfolio: details popup window
- I use a JavaScript array to store all of the information for all projects.
- For each project I store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- I update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads
-

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -11,6 +12,109 @@ function menuClickedHandler() {
...
+      <li class="tags">Ruby</li>
+      <li class="tags">Bootstrap</li>
+  
+    </ul>
+   <div class="live-btn">
+    <button  class="btn" type=button"">See live <img class="live-icon" src="icons/liveIcon.svg" alt=""> </button>
+    <button  class="btn" type="button">See Source <img class="live-icon" src="icons/buttonIconGitHub.svg" alt=""></button></div>
+  </div>
+  </div>
+  </div>`;
```
Comment (AshakaE) [inline index.js:102]: - [ ] Kindly work on the image responsiveness, it looks deformed on desktop, you can use a max width to make sure the modal does not exceed that width on desktop screens
image

**Review by AshakaE — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good job setting up project linters 👍 
- Proper use of Github flow ✅ 
- Dynamic project rendering 👏 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @HenaryKhuraijam 

Your project is complete! Great job t 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- design ok
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (HenaryKhuraijam) [reply index.js:102]: Thank you finding out my mistakes

**Comment by HenaryKhuraijam** [comment]
Make changes as suggested

**Pull request merged**